doc/getting_started/installation_linux.rst

```diff
@@ -164,23 +179,30 @@ Follow these steps to install the SDK on your Linux host system.
       export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
 
-  To use the same toolchain in new sessions in the future, you can set the
-  variables in the file :file:`${HOME}/.zephyrrc`, for example:
+.. _sdkless_builds:
 
-  .. code-block:: console
+Building on Linux without the Zephyr SDK
+****************************************
 
-     cat <<EOF > ~/.zephyrrc
-     export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
-     export ZEPHYR_SDK_INSTALL_DIR=/opt/zephyr-sdk
-     EOF
+The Zephyr SDK is provided for convenience and ease of use. It provides
+toolchains for all Zephyr target architectures, and does not require any extra
+flags when building applications or running tests. In addition to
+cross-compilers, the Zephyr SDK also provides prebuilt host tools. It is,
+however, possible to build without the SDK's toolchain by using another
+toolchain as as described in the main :ref:`getting_started` document.
 
-  .. note::
-     Use ``<sdk installation directory>`` in place of ``/opt/zephyr-sdk/`` in the
-     above shown example if the SDK installation location is not default.
+As already noted above, the SDK also includes prebuilt host tools.  To use the
+SDK's prebuilt host tools with a toolchain from another source, keep the
+ZEPHYR_SDK_INSTALL_DIR environment variable set to the Zephyr SDK installation
```

Comment:
```suggestion
``ZEPHYR_SDK_INSTALL_DIR`` environment variable set to the Zephyr SDK installation
```

Reply:
Would you prefer to use `:envvar:\`ZEPHYR_SDK_INSTALL_DIR\`` instead?

http://www.sphinx-doc.org/en/1.5.2/markup/inline.html#role-envvar

Similarly for ZEPHYR_SDK_INSTALL_DIR and CROSS_COMPILE below.

I'm going to assume the answer is "yes", please let me know otherwise and I'll go back to straight double backticks.